cumulusci/tasks/salesforce/package_upload.py

```diff
@@ -59,6 +70,43 @@ def _run_task(self):
             self._set_dependencies()
             self._log_package_upload_success()
 
+    """This functions validates the major version and minor version if passed and sets them in the options dictionary if not passed."""
+
+    def _validate_versions(self):
+
+        """Fetching the latest major,minor and patch version from Database"""
+        version = self._get_one_record(
+            (
+                "SELECT MajorVersion,"
+                "MinorVersion,"
+                "PatchVersion,ReleaseState FROM MetadataPackageVersion Order by MajorVersion Desc,MinorVersion Desc, PatchVersion Desc LIMIT 1"
```

Comment:
```suggestion
            """
            SELECT 
              MajorVersion, 
              MinorVersion, 
              PatchVersion,
              ReleaseState
            FROM MetadataPackageVersion
            ORDER BY
              MajorVersion DESC, 
              MinorVersion DESC, 
              PatchVersion DESC
            LIMIT 1
            """
```

Reply:
Triple-quoted strings can also be used for multiline literals in Python. It's an option here to help with readability.